CMake/cdat_modules_extra/setup_runtime.sh.in

```diff
@@ -10,20 +10,21 @@ function cleanup {
 # which can lead to errors.
 if [ -n "${UVCDAT_SETUP_PATH}" ] ; then
   if [ "${UVCDAT_SETUP_PATH}" = "${install_prefix}" ] ; then
+    echo "Nothing to do since UVCDAT is already setup at: ${UVCDAT_SETUP_PATH}" 1>&2
     cleanup
     return 0
   else
-    echo "ERROR: UVCDAT setup was previously sourced at \"${UVCDAT_SETUP_PATH}\"" 1>&2
-    echo "ERROR: There is no need to run setup_runtime manually anymore." 1>&2
-    echo "ERROR: Open a new shell in order to use a different install location." 1>&2
+    echo "INFO: UVCDAT setup was previously sourced at: ${UVCDAT_SETUP_PATH}" 1>&2
+    echo "INFO: There is no need to run setup_runtime manually anymore." 1>&2
+    echo "INFO: Open a new shell in order to use a different install location." 1>&2
```

Comment:
You run the script for one reason, that is to load that UV-CDAT version in your shell. If you can't load it, it is as fatal an error as it can be. While would you pretend otherwise?

The only place the error status matters is when run from another script. If you run this from another script and encounter the condition, something is very wrong. Why would you possibly want to hide that?

I get that you like picking on whatever I say, but breaking the current behavior on purpose just because I wrote it is childish. UNIX has a notion of error codes, why not use it? Sure, you can figure it out by reading the terminal messages, but seriously, why not report errors as errors?